docs/usage/creating.md

```diff
@@ -0,0 +1,32 @@
+# Creating Projects and Simulations
+
+## Projects
+
+Projects contain simulations of a certain type. Go to `/` or click on the HPC-Cloud logo to go to the projects page. 
+
+### Creating 
+
+Click on the "+" icon on the top right in the toolbar, this will take you to a new page. On this page name and describe your project. Depending on the project type, you may also need to upload some files required for simulations. Name your project and add a description if desired. Click "Create Project" and after your file(s) upload you'll be taken to the simulations page for your new project. 
+
+### Editing 
+If you named or described your project incorrectly click on the pen icon on the homepage and modify these attributes. You cannot reupload files if the project required any.
+
+### Deleting
+ 
+You can only delete projects which have no simulations associated with them. Click on the pen icon for a project and you'll be taken to the edit page. Clicking the "Delete project" button will delete the project, you will be prompted beforehand. 
+
+## Simulations
+
+Simulations represent collections of taskflows depending on the type of parent project.
```

Comment:
I think that is more like the Simulation _Step_, but it's also as you said. I'm thinking specifically when you "create simulation" you're making a set of steps/taskflows, one of which is the actual simulation which runs modeling code. Perhaps some term revision could be in order to distinguish Simulation, Simulation Step, and _Simulation_ (which runs modeling code)?